examples/bert/src/bert_layers.py

```diff
@@ -434,21 +442,20 @@ def get_slopes_power_of_2(n_heads: int) -> List[float]:
                         2 * closest_power_of_2)[0::2][:n_heads -
                                                       closest_power_of_2]
 
-        max_token_length = config.max_position_embeddings
-        context_position = torch.arange(max_token_length)[:, None]
-        memory_position = torch.arange(max_token_length)[None, :]
+        context_position = torch.arange(max_token_length, device=device)[:,
+                                                                         None]
```

Comment:
Is the extra line here intentional?

Reply:
Sigh. No, it's the auto formatter being a bully. I might try to see if I can outsmart it because this is nuts.

Reply:
Aha! Problem solved.

Reply:
> auto formatter

BOOOOOOOOO